charts/kuberpult/run-kind.sh

```diff
@@ -298,6 +298,7 @@ argocd:
   token: "$token"
   server: "https://argocd-server.${ARGO_NAMESPACE}.svc.cluster.local:443"
   insecure: true
+  refreshEnabled: true
```

Comment:
https://github.com/freiheit-com/kuberpult/blob/8cb4e6e1d6b08f57329a343279b52f3576fddd29/services/frontend-service/pkg/cmd/server.go#L465

"Unimplemented" is not accurate, right? It's just not enabled.
Maybe "424"? https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/424

Reply:
or maybe just change the message to 
"rollout status not enabled"

Reply:
I created a follow up issue in github.